src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sumA = 0;
+
+  const sum = (a) => {
+    if (a !== undefined) {
+      sumA += a;
+
+      return sum;
+    } else {
```

Comment:
```suggestion
    } 
```
`else` is redundant here. You are returning something in both cases